e2e/scenario/formauth_idp.conf.js

```diff
@@ -0,0 +1,10 @@
+exports.config = {
+    auth: {
+        'sapcloud-form': {
+            user: 'user',
+            pass: 'pass',
+            idpSelector: "#saml-login-link",
+            redirectUrl: /app.html/
```

Comment:
why do you need the redirectUrl here ?

Reply:
the test starts at login.html, and then redirects to app.html

Reply:
actully the test starts on app.html, it does the redirect to login.html that redirects back.